src/hooks/useTokenAllowance.ts

```diff
@@ -48,7 +48,8 @@ export function useUpdateTokenAllowance(
       if (!contract) throw new Error('missing contract')
       if (!spender) throw new Error('missing spender')
 
-      const allowance = MaxUint256.toString()
+      const maxAllowance = MaxUint256.toString()
```

Comment:
This is static. Could you bring it to the top of the file as `const MAX_ALLOWANCE = MaxUint256.toString()`?